graphql/executor.go

```diff
@@ -142,6 +145,15 @@ type resolveAndExecuteCacheKey struct {
 }
 
 func (e *Executor) resolveAndExecute(ctx context.Context, field *Field, source interface{}, selection *Selection) (interface{}, error) {
+	var span opentracing.Span
+	if _, ok := field.Type.(*Scalar); !ok {
+		innerSpan, innerCtx := opentracing.StartSpanFromContext(ctx, fmt.Sprintf("thunder.graphql.resolve(%s)", selection.Alias))
```

Comment:
Code similar to this repeats a lot. Could we simplify / extract some logic into a helper, or is this the way it's going to be?